src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -73,6 +74,9 @@ public LoanRepository(Clients clients) {
 
     if(newLoanAndRelatedRecords.getLoanPolicy() != null) {
       storageLoan.put("loanPolicyId", newLoanAndRelatedRecords.getLoanPolicy().getId());
+      if (Objects.nonNull(newLoanAndRelatedRecords.getLoanPolicy())) {
```

Comment:
Why is the loan policy name being stored?

There is no `loanPolicyName` defined for the [loan in storage](https://github.com/folio-org/mod-circulation-storage/blob/master/ramls/loan.json). Hence, this change would result in a runtime error and being unable to save loans.